### PR TITLE
[Fix] move target_wrapper into utils

### PR DESCRIPTION
**Pull request by grimoire**
## Motivation

Fix for https://github.com/open-mmlab/mmdeploy/issues/13

## Modification

move `target_wrapper` into utils

### CODE REVIEW

File: mmdeploy/utils/utils.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import logging
+import sys
+import traceback
+
+
+def target_wrapper(target, log_level, ret_value, *args, **kwargs):
```
Comment (RunningLeon) [inline mmdeploy/utils/utils.py:6]: may need to add docstring and typehint.

**Comment by AllentDan** [comment]
need a unit test?

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by hhaAndroid — DISMISSED** [review]

**Review by AllentDan — DISMISSED** [review]

**Review by AllentDan — APPROVED** [review]

**Review by hhaAndroid — APPROVED** [review]

File: tests/test_utils/test_util.py [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 
 import mmcv
 import pytest
+import torch.multiprocessing as mp
 
 import mmdeploy.utils as util
 from mmdeploy.utils.constants import Backend, Codebase, Task
 from mmdeploy.utils.export_info import dump_info
 from mmdeploy.utils.test import get_random_name
+from mmdeploy.utils.utils import target_wrapper
```
Comment (RunningLeon) [inline tests/test_utils/test_util.py:15]: `from mmdeploy.utils import target_wrapper`

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**